docs/docs/guides/configure-command.md

```diff
@@ -0,0 +1,339 @@
+| `--delete <id>` | Delete a saved configuration |
+| `--export <id>` | Export a saved configuration to file |
+
+## Interactive Walkthrough
+
+### Creating a New Configuration
+
+```
+┌   Saiki Agent Configuration
+│
+◇  Choose your LLM provider
+│  ● OpenAI
+│  ○ Anthropic  
+│  ○ Google
+│  ○ Groq
+│
+◇  Enter the model name for openai
+│  gpt-4o-mini
+│
+◇  How do you want to handle the openai API key?
+│  ● Use environment variable (Will use $OPENAI_API_KEY)
+│  ○ Enter manually (Not recommended for production)
+│  ○ Skip for now (Configure later)
+│
+◇  How would you like to choose MCP servers?
+│  ● Choose a preset (Recommended - common combinations)
+│  ○ Browse by category (Select from organized categories)
+│  ○ Select individually (Full control over selection)
+│  ○ Skip MCP servers (Add them later)
+│
+◇  Choose a preset configuration
+│  ● Essential Tools (Filesystem + Web browsing)
+│  ○ Developer Setup (Essential + GitHub + Terminal)
+│  ○ Productivity Suite (Essential + Notion + Slack)
+│  ○ Data & Analytics (Essential + Database tools)
+│
+◇  Customize the system prompt?
+│  ○ Yes / ● No
+```
+
+### Loading an Existing Configuration (Improved!)
+
+```
+┌   Saiki Agent Configuration
+│
+◆  Loaded configuration: My Development Agent
+│
+◇  Current LLM provider: openai. Change?
+│  ● Keep openai
+│  ○ OpenAI
+│  ○ Anthropic
+│  ○ Google
+│  ○ Groq
+│
+◇  Current model: gpt-4o-mini. Press Enter to keep, or type new model:
+│  gpt-4o-mini
+│
+◇  You have 3 MCP server(s) configured. What would you like to do?
+│  ● Keep current servers (No changes to MCP servers)
+│  ○ Modify server selection (Add or remove servers)
+│  ○ Replace all servers (Start fresh with server selection)
+│
+◇  Modify the system prompt?
+│  ○ Yes / ● No
+```
+
+### Smart MCP Server Presets
+
+Choose from optimized combinations:
+
+- **Essential Tools**: filesystem + puppeteer (web browsing)
+- **Developer Setup**: Essential + github + terminal 
+- **Productivity Suite**: Essential + notion + slack
+- **Data & Analytics**: Essential + sqlite + postgres
+
+## Generated Configuration Files
+
+Configurations are automatically saved to `agents/` directory with complete YAML:
+
+```yaml
+# Saiki Agent Configuration
+# Generated on 2024-01-15T10:30:00.000Z
+
+systemPrompt: "You are a helpful AI assistant with access to tools. Use these tools when appropriate to answer user queries. You can use multiple tools in sequence to solve complex problems."
+
+mcpServers:
+  filesystem:
+    type: stdio
+    command: npx
+    args:
+      - "-y"
+      - "@modelcontextprotocol/server-filesystem"
+      - "/allowed/path"
+    connectionMode: lenient
+  
+  puppeteer:
+    type: stdio
+    command: npx
+    args:
+      - "-y"
+      - "@modelcontextprotocol/server-puppeteer"
+    connectionMode: lenient
+
+llm:
+  provider: openai
+  model: gpt-4o-mini
+  apiKey: ${OPENAI_API_KEY}
+```
+
+## Configuration Lifecycle
+
+### 📝 **Creating New Configurations**
```

Comment:
let's use the term Agent Configurations instead of just Configurations everywhere

since we moved to `--agent` it is more aligned